python/deptry/core.py

```diff
@@ -14,7 +14,7 @@
 from deptry.exceptions import IncorrectDependencyFormatError, UnsupportedPythonVersionError
 from deptry.imports.extract import get_imported_modules_from_list_of_files
 from deptry.module import ModuleBuilder, ModuleLocations
-from deptry.python_file_finder import PythonFileFinder
+from deptry.python_file_finder import get_all_python_files_in
```

Comment:
Is it an idea to directly use the function that is defined in Rust here, and remove `python_file_finder.py` completely?

```suggestion
from deptry.rust import find_python_files
```

Reply:
Yeah I must say that this is originally what I wanted to do, but it made tests more complex as they would need to instantiate `Core` class with a load of arguments we don't need to, as there is no default for the arguments. Eventually, I'd like to rework a bit the `Core` class, which right now feels like a god class with a too-generic name.